data/lib/src/repository/authentication/token_repository_impl.dart

```diff
@@ -46,6 +46,14 @@ class TokenRepositoryImpl extends TokenRepository {
     await sharedPreferences.setObject(keyPermanentToken, token.toPermanentToken().toJson());
   }
 
+  @override
+  Future removeToken() async {
+    if (!(await getToken()).isTokenValid()) {
+      throw NoTokenStored();
+    }
```

Comment:
What is the intent of this line?

Reply:
Wanted to throw an exception if no token is stored but maybe overkill

Reply:
Should I remove it ?

Reply:
yes, I think so, No token is ok for the case delete it ^^